app/models/artist.rb

```diff
@@ -128,6 +128,14 @@ def doing_open_studios?(key = nil)
 
   alias doing_open_studios doing_open_studios?
 
+  def remind_for_open_studios_register?
+    current_event = OpenStudiosEventService.current
+    current_event_start_date = current_event.start_date
+    time_to_remind = Time.zone.now < current_event_start_date && current_event_start_date <= Time.zone.now + 12.weeks
```

Comment:
I think this `Time.zone.now < current_event_start_date` should always be true because (I think) the way we compute the "current" open studios is "it is in the next available OS in the future".  So that clause is probably not necessary.

Reply:
in the definition of `current`[ here](https://github.com/bunnymatic/mau/blob/93b8c80aa6b685031ef6eb70043d27ce90b7beb2/app/models/open_studios_event.rb#L35): , it is `future.first || past.last`. So I assumed that there would always be some current event even if it's passed already

Reply:
oh true.  i guess that's a good check.  i didn't look but assumed that `current` was more like `future.first || nil`.